lib/circuitry/subscriber.rb

```diff
@@ -141,7 +157,7 @@ def process_messages_synchronously(messages, &block)
     end
 
     def process_message(message, &block)
-      message = Message.new(message)
+      # message = Message.new(message)
```

Comment:
why is this commented out?

Reply:
Oh, it should be gone.

So the idea right now is that the `message_parser` goes through all the `messages` and wraps them with the `Message` class, instead of doing it one by one when the message gets passed to `process_message`

This allows for the messages being filtered in the new `filter_with` option, to be `Circuitry::Message` instances rather than plain json

Reply:
aaaaah makes sense!